### PR TITLE
querty = cool

### PR DESCRIPTION
**Pull request by querty27**
pull this or else

### CODE REVIEW

**Comment by querty27** [comment]
yo i love this pr

**Comment by LucasJWilliams** [comment]
Could you move the command to `misc.py` please. A decorator to check for the staff role would be useful to prevent any random in chat from spamming it.

File: src/cogs/modmail.py [hunk 1]
```diff
@@ -194,6 +194,16 @@ async def info_button(self, ctx: ComponentContext):
     # Other commands
     # --------------------------------------------
 
+    @commands.command()
```
Comment (LucasJWilliams) [inline src/cogs/modmail.py:197]: has_role decorator here

**Comment by LucasJWilliams** [comment]
Added the command, `range(10)` I have changed to `range(4)` because it's the most the API would let me get away with in one go.

**Pull request closed without merging**